app/src/main/java/com/example/myapplication/Constants.java

```diff
@@ -8,10 +8,10 @@
     static final String ORDERS = "OpenOrders";
     static final String MANOT_SUBCOLLECTION = "Manot";
     static final String MONEY_MADE = "הכסף שנצבר: %s מתוך 70 שקלים";
-    static final String LOCKED_TEXT = "נעול";
+    static final String LOCKED_TEXT = "הזמן!";
```

Comment:
Should reformat the constane reference. Myabe "READY_TEXT"?

Reply:
You're right, I'll take care of it after merging.